app/assets/javascripts/plate-map/create-field.js

```diff
@@ -1090,6 +1128,39 @@ var plateLayOutWidget = plateLayOutWidget || {};
           }
           that.fieldWarningMsg(field, warningText, mainFieldWarning);
         };
+
+        field.parseMainFieldVal = function(val) {
+          var optMap = field.data.options;
```

Comment:
You are iterating over field.data.options (which is an array) as if it is an object. Use: 

```
for (var idx = 0; idx < optMap.length; idx++) {
```

Reply:
Resolved